src/pawns.cpp

```diff
@@ -148,8 +148,8 @@ namespace {
         // Score this pawn
         if (support | phalanx)
         {
-            int v =  Connected[r] * (4 + 2 * bool(phalanx) - 2 * bool(opposed) - bool(blocked)) / 2
-                   + 21 * popcount(support);
+            int v =  Connected[r] * (4 + 2 * bool(phalanx) - 2 * bool(opposed)) / 2
```

Comment:
The divide by 2 can now also be removed.
```int v =  Connected[r] * (2 + bool(phalanx) - bool(opposed))```

Reply:
@mstembera wudn't that change the final computed value for `v`, why then can it be removed?

Reply:
@anshulongithub 
(4 + 2x - 2y) / 2 is the same as (2+x- y)

Reply:
oh yeah! thank you so much @Rocky640, I shud have read the equation more carefully. 
Thank you.